src/contexts/_default.json

```diff
@@ -3,45 +3,27 @@
   "base": "_base",
   "layers": [
     {
-      "title": "MSP DESSERTE MUN 911",
+      "title": "WFS",
```

Comment:
Drag this to another context for the simple-feature-list and keep default as it is

Reply:
Created a new context